ApiFirst/AppServices/Services/CustomerBankInfoAppService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AppServices.Interfaces;
+using DomainServices.Interfaces;
+using System;
+using System.Threading.Tasks;
+
+namespace AppServices.Services
+{
+    public class CustomerBankInfoAppService : ICustomerBankInfoAppService
+    {
+        private readonly ICustomerBankInfoService _customerBankInfoServices;
+
+        public CustomerBankInfoAppService(ICustomerBankInfoService customerBankInfoServices)
+        {
+            _customerBankInfoServices = customerBankInfoServices ??
+                throw new ArgumentNullException(nameof(customerBankInfoServices));
+        }
+
+        public async Task CreateAsync(long customerId)
+        {
+            await _customerBankInfoServices.CreateAsync(customerId);
```

Comment:
Em todos os métodos dessa classe
```suggestion
            await _customerBankInfoServices.CreateAsync(customerId).ConfigureAwait(false);
```